action.yml

```diff
@@ -2,5 +2,5 @@ name: "Run OpenPedCan Analysis"
 description: "Run analysis workflow"
 runs:
   using: "docker"
-  image: "Dockerfile"
+  image: "pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:analysisjob"
```

Comment:
Using a pre-built image like this in the custom workflow also sets us up to put the analysis job downstream from the build and push workflow per https://github.com/PediatricOpenTargets/ticket-tracker/issues/393.